helm/nessie/values.yaml

```diff
@@ -14,6 +14,7 @@ versionStoreType: INMEMORY
 
 # this is needed when selecting ROCKS
 rocksdb:
+  storageClassName: standard
   storageSize: 1Gi
```

Comment:
@julwin : This PR adds storage class as a settable parameter as you suggested. `storageSize` was made already available by the previous PR.

Reply:
Sounds good 😄